ml-agents/mlagents/trainers/tests/test_bc.py

```diff
@@ -78,15 +78,17 @@ def test_bc_trainer_add_proc_experiences(dummy_config):
     # Test add_experiences
     returned_braininfo = env.step()
     trainer.add_experiences(
-        returned_braininfo, returned_braininfo, {}
+        returned_braininfo["Ball3DBrain"], returned_braininfo["Ball3DBrain"], {}
```

Comment:
I know it's test code, so the quality bar is lower. but maybe do e.g. `brain_name = "Ball3DBrain"` and use that instead of the string everywhere?

Reply:
I'll fix this now. Thanks.